packages/components/psammead-embed-error/src/index.jsx

```diff
@@ -1,73 +1,93 @@
 import React from 'react';
-import { string, bool } from 'prop-types';
+import { string, bool, shape } from 'prop-types';
 import styled from 'styled-components';
-import { C_CHALK, C_EBON } from '@bbc/psammead-styles/colours';
 import {
-  GEL_SPACING_TRPL,
-  GEL_SPACING_HLF,
-} from '@bbc/gel-foundations/spacings';
-import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+  C_LUNAR,
+  C_SHADOW,
+  C_EBON,
+  C_METAL,
+} from '@bbc/psammead-styles/colours';
+import { GEL_SPACING_TRPL, GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { getSansRegular, getSansBold } from '@bbc/psammead-styles/font-styles';
 import { GEL_BODY_COPY } from '@bbc/gel-foundations/typography';
-import { BBC_BLOCKS, coreIcons } from '@bbc/psammead-assets/svgs';
+import { BBC_BLOCKS } from '@bbc/psammead-assets/svgs';
 
 const GOLDEN_RATIO_PERCENT = '38.2%';
-const GEL_SPACING_TRPL_MINUS_HLF = '1.25rem';
-const GEL_SPACING_QRT = '0.125rem';
+const BBC_BLOCKS_WIDTH = '10rem';
+const FAUX_BBC_BLOCKS_SPACE = '6rem';
+
+const FILL_VIEWPORT_STYLES = `
+  background-position: center center;
```

Comment:
```suggestion
  background-position: center;
```

Reply:
@FK78, we'll also need to update snapshots for this change.